File: Java/PageObject/05_TestBase/TestBase.java
package 05_TestBase;

import org.junit.After;
import org.junit.Before;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class TestBase {
	
	WebDriver driver;
	
	@Before
	public void SetUp() {
		driver = new FirefoxDriver();
		driver.manage().window().maximize();
		driver.get("http://www.mercadolibre.com.ar/");
	}
	
	@After
	public void TearDown() {
		driver.quit();
	}
}

File: Python/page_object/06_page_base/test_base.py
import unittest
from selenium import webdriver


class BaseTestCase(unittest.TestCase):
    def setUp(self):
        self.driver = webdriver.Firefox()
        self.driver.implicitly_wait(30)
        self.driver.maximize_window()

        self.driver.get("http://www.mercadolibre.com.ar/")

    def tearDown(self):
        self.driver.quit()
